front/src/modules/simulationResult/components/SpaceTimeChart/useGetProjectedTrainOperationalPoints.ts

```diff
@@ -30,7 +36,12 @@ const useGetProjectedTrainOperationalPoints = (
 
   useEffect(() => {
     const getOperationalPoints = async () => {
-      if (infraId && pathfindingResult && pathfindingResult.status === 'success') {
+      if (
+        infraId &&
+        trainScheduleUsedForProjection &&
+        pathfindingResult &&
+        pathfindingResult.status === 'success'
```

Comment:
you can combine those in `pathfindingResult?.status === 'success'`

Reply:
Done